src/test/java/org/broadinstitute/hellbender/tools/walkers/SplitIntervalsIntegrationTest.java

```diff
@@ -84,33 +86,49 @@ public void testSingleScatter() {
     public void testTwoIntervals() {
         final int scatterCount = 5;
         final File outputDir = createTempDir("output");
-        final String[] args = {
-                "-L", "20:1000000-2000000",
-                "-L", "20:3000000-4000000",
-                "-R", REFERENCE.toAbsolutePath().toString(),
-                "-" + SplitIntervals.SCATTER_COUNT_SHORT_NAME, Integer.toString(scatterCount),
-                "-O", outputDir.getAbsolutePath()
-        };
+        final ArgumentsBuilder args = new ArgumentsBuilder()
```

Comment:
I believe this causes line 19 and 20 to no longer be needed.
import java.nio.file.Path;
import java.nio.file.Paths;

Reply:
done